client/src/components/content/HeaderNavbar/HeaderNavbar.jsx

```diff
@@ -11,34 +11,51 @@ const { Header } = Layout;
  * @description Renders the header navbar for all pages and initially selects the passed key-element.
  * @example return <HeaderNavbar selectedKey={2} />
  */
-const HeaderNavbar = (props) => (
-  <Header>
-    <Menu
-      theme='dark'
-      mode='horizontal'
-      // eslint-disable-next-line react/prop-types
-      defaultSelectedKeys={[props.selectedKey.toString()]}
-    >
-      <Link to='/'>
-        <img
-          style={{ margin: '0 1rem 0 -1rem', height: '3rem' }}
-          src={logoCTA}
-          alt='ark_automate Icon'
-        />
-      </Link>
-      <Menu.Item key='1'>
-        Overview
+const HeaderNavbar = (props) => {
+  const { selectedKey } = props;
+  let onOverview = true;
+  if (selectedKey === 1) {
```

Comment:
What does this do? When is selectedKey === 1?

Reply:
It is 1 when you are on the robot overview page. This takes care of "hiding" the modeler and robot code tabs when you are on the overview page